restler/unit_tests/test_basic_functionality_end_to_end.py

```diff
@@ -291,6 +291,32 @@ def test_payload_body_checker(self):
         except TestFailedException:
             self.fail("Payload body failed: Garbage Collector")
 
+    def test_payload_body_checker_advanced(self):
+        """ This checks that the payload body checker sends all of the correct
+        requests in the correct order for more complicated bodies (including an array body)
```

Comment:
>in the correct order [](start = 17, length = 20)

Nice!  Could you please elaborate in this comment specifically what about orderings the new test covers?